Packs/GCP/Integrations/GCP/GCP.py

```diff
@@ -1471,6 +1478,267 @@ def get_credentials(args: dict, params: dict) -> Credentials:
         raise DemistoException(f"Failed to authenticate with GCP: {str(e)}")
 
 
+def gcp_compute_network_get_command(creds: Credentials, args: dict[str, Any]) -> CommandResults:
+    """
+    Returns the specified network.
+    Args:
+        creds (Credentials): GCP credentials with admin directory security scope.
+        args (dict[str, Any]): _description_
+    Returns:
+        CommandResults: _description_
+    """
+    project = args.get("project_id")
+    network = args.get("network")
+
+    compute = GCPServices.COMPUTE.build(creds)
+    response = compute.networks().get(project=project, network=network).execute()
+
+    data_res = {"name": response.get("name"), "id": response.get("id")}
+
+    headers = ["id", "name"]
+
+    readable_output = tableToMarkdown(f"GCP network {network}", data_res, headers=headers, removeNull=True)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="GCP.Compute.Networks",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+    )
+
+
+def gcp_compute_image_get(creds: Credentials, args: dict[str, Any]) -> CommandResults:
+    """
+    Returns the specified image.
+    Args:
+        creds (Credentials): GCP credentials.
+        args (dict[str, Any]): _description_
+    Returns:
+        CommandResults: _description_
+    """
+    project = args.get("project_id")
+    image = args.get("image")
+
+    compute = GCPServices.COMPUTE.build(creds)
+    response = compute.images().get(project=project, image=image).execute()
+
+    data_res = {"id": response.get("id"), "name": response.get("name")}
+    headers = ["id", "name"]
+
+    readable_output = tableToMarkdown(f"GCP images {image}", data_res, headers=headers, removeNull=True)
```

Comment:
```suggestion
    data_res = {"id": response.get("id"), "name": response.get("name")}
    headers = ["id", "name"]

    readable_output = tableToMarkdown(f"GCP images {image}", data_res, headers=headers, removeNull=True, headerTransform=pascalToSpace)
```
Add the creationTimestamp and description to the HR.